templates/shared/forms/interactive/embedding.html

```diff
@@ -1,124 +1,257 @@
 <div class="p-modal" id="contact-modal">
-  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
-    <header class="p-modal__header" style="display: block; border-bottom: 0;">
-      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
-      <h2 class="p-modal__title u-sv1" id="modal-title">Let's get in touch</h2>
+  <div class="p-modal__dialog is-wide-modal"
```

Comment:
Unless the pdm specified differently you should only include the fields that appear in [the copydoc](https://docs.google.com/document/d/1FTyjPUtLEBvz_-MzNEDPbTWuSrr8NavzUmBcVBCG4r8/edit?tab=t.0)